common/CMakeLists.txt

```diff
@@ -193,3 +199,4 @@ PCL_ADD_INCLUDES("${SUBSYS_NAME}" impl ${impl_incs})
 PCL_ADD_INCLUDES("${SUBSYS_NAME}" console ${tools_incs})
 PCL_ADD_INCLUDES("${SUBSYS_NAME}" range_image ${range_image_incs})
 PCL_ADD_INCLUDES("${SUBSYS_NAME}" range_image/impl ${range_image_incs_impl})
+PCL_ADD_INCLUDES("${SUBSYS_NAME}" execution ${executor_incs})
```

Comment:
I'd propose either a separate module or inside `common`. The includes could be via `<pcl/common/executor.h>` instead of `<pcl/common/execution/executor.hpp>`

Sole reason: Multiple sub-module aren't handled well in PCL's CMake